### ISSUE TITLE
Monthly Chat Agenda FIXME (2023-06-05 and 2023-06-19)

### ISSUE THREAD

**Issue by RedXanadu** [coreruleset/coreruleset#3221]
This is the Agenda for the two Monthly CRS Chats.

The general chat is going to happen on https://owasp.slack.com in the channel #coreruleset on Monday, 2023-06-05, at 20:30 CET. That's the 1st Monday of the month. A separate issue chat is happening at the same location, same time on Monday, 2023-06-19. That's the 3rd Monday of the month. Please note that we have a [CRS calendar](https://calendar.google.com/calendar/u/0/embed?src=c_glsll5etlmcb3it0k433fcl5rc@group.calendar.google.com) (maintained by @fzipi).

Archived previous meetings and their decision are [here](https://github.com/coreruleset/project-chat-archive/blob/master/README.md).

### Outside development

* [FIXME](FIXME)

#### Rules

* FIXME: Please fill in

#### CRS Sandbox

* FIXME: Please fill in

#### CRS Bug Bounty and Security

* FIXME: Please fill in

#### Plugins

* FIXME: Please fill in

#### Documentation and Public Relations

* FIXME: Please fill in

#### Project Administration and Sponsor relationships

* FIXME: Please fill in

#### Tools

* FIXME: Please fill in

#### Testing incl. Seaweed and many future plans

* FIXME: Please fill in

#### Containers

* FIXME: Please fill in

#### CRS Status Page

* FIXME: Please fill in

## Project discussions and decisions

* *Item carried over from April and May meetings: agreed to formally add to the agenda for discussion at the June meeting.*
  **Coraza/libcoraza:** Should the CRS project pay for someone to build an Nginx connector?
  * We were waiting on the imminent milestone release of Coraza due out in May: Coraza v3.
* *Item carried over from second May meeting:*
  PR #3202 **->**
  * Is this really what we want?
  * How to maintain this - fix future FPs?
  * Do we do a tag? Which one?

* FIXME: Please fill in

### PRs that have been merged since the last meeting

* #FIXME

We merged FIXME PRs since the last monthly project chat.

### Open issues and PRs

* As of Monday, we have FIXME open issues.
* As of Monday, we have FIXME open pull requests.

# Separate 2nd Meeting (Monday, 2023-06-19)

* FIXME: Agenda to be defined

## How to get to our slack and join the meeting?

If you are not yet on the OWASP Slack, here is your invite: https://owasp.org/slack/invite .

Everybody is welcome to join our community chat.

**Comment by franbuehler** [comment]
## Decisions June 5th

* Proposal to rename the project from `OWASP ModSecurity Core Rule Set` to `OWASP WAF Core Rule Set` or simply to `OWASP Core Rule Set` by @fzipi. This will be more in line that we support other engines. **->** We all agree on the change, but we're not doing it now. We'll continue the conversation during the next few weeks how to get there.
* *Item carried over from April and May meetings: agreed to formally add to the agenda for discussion at the June meeting.*
  **Coraza/libcoraza:** Should the CRS project pay for someone to build an Nginx connector?
  * We were waiting on the imminent milestone release of Coraza due out in May: Coraza v3.
* *Item carried over from second May meeting:*
  PR #3202
  * Is this really what we want?
  * How to maintain this - fix future FPs?
  * Do we do a tag? Which one?
* Roadmap to CRS v4 release
* v3.3.5. We agreed at the February meeting to do a 3.3.5 patch-based fix release with the scoring/PL fixes, along with a blog post.
  * Can we agree on the messaging that we're happy to publish to users, vendors, etc. (i.e. "we will not provide a formal fix release because \<reason\>") so that the blog post writing can begin now? Are we happy to just say "we don't have time to do a formal release"?
* #3219: Do we want to include 127.0.0.1 and friends into the into ssrf rule?
* :memo: Meeting agendas: GitHub issue editing is not very robust when there are multiple users… We're having problems whereby CRS team members accidentally overwrite each other's edits (and even overwrite their own edits!) when adding things to a meeting agenda GitHub issue. Even refreshing the page first doesn't always help. Is there a better system for us to use?
  * Collaborate in a Google Doc? Then we copy-paste it into a GitHub issue just before the meeting?
  * Something else?

**Comment by franbuehler** [comment]
## Decisions June 19

* Talk about PHP function rules overhaul: https://github.com/coreruleset/coreruleset/pull/3228
  This is the last major rules construction site before we can tackle the release candidate.
:large_blue_circle: **Decision:** @theseion will do a new spell.sh that no longer uses spell, but a new and modern tool. This PR will just be a quick solution. In CRS 4.1 this could be rewritten again and then be added to crs-toolchain.
:large_blue_circle: **Decision:** @M4tteoP will work with @dune73 to put everything into a new PR.
* https://github.com/coreruleset/plugin-registry/pull/13: exclusion rules plugin for RoundCube -> 3 separate plugins?
* Use GitHub wiki for agenda (https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages#adding-or-editing-wiki-pages-locally)?
